> I added [this patch](https://github.com/randombit/botan/pull/1884) to the recipe. Lets see if the old Xcodes like it...

It still fails on macOS with Xcode 7.3 (that came out more than three years ago), due to the missing support for C++11's `thread_local` keyword in this compiler:
```
botan_all.cpp:13216:11: error: thread-local storage is not supported for the current target
```

The `thread_local` keyword is currently used in [a single location](https://github.com/randombit/botan/blob/master/src/lib/utils/cpuid/cpuid.cpp#L19) only and could potentially be patched, if support for this version is still required. However, I'd argue for dropping support for Xcode 7.3.

@SSE4 What do you think?

--- a/.travis.yml
+++ b/.travis.yml
@@ -49,9 +49,6 @@ matrix:
       - <<: *linux
         env: CONAN_CLANG_VERSIONS=7.0 CONAN_DOCKER_IMAGE=conanio/clang7 CONAN_ARCHS=x86_64
       - <<: *osx
-        osx_image: xcode7.3
-        env: CONAN_APPLE_CLANG_VERSIONS=7.3
-      - <<: *osx
         osx_image: xcode8.3
         env: CONAN_APPLE_CLANG_VERSIONS=8.1
       - <<: *osx

